.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "spec"]
+	path = spec
```

Since we're only using purl-spec test data, we should keep it inside `tests/data/`.
```suggestion
	path = tests/data/purl-spec/
```

---

I guess we should not keep the complete spec in tests